Nos.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -51,8 +51,7 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/nostr-sdk/nostr-sdk-ios",
       "state" : {
-        "revision" : "afbfc4b3c347a72f9d467a6e213d22fa66a2f63e",
-        "version" : "0.1.0"
+        "revision" : "8968ec00caa51d03d48bd2e91e70f121950fa05d"
```

Comment:
We can point to a version number once they bump to the change introduced through https://github.com/nostr-sdk/nostr-sdk-ios/pull/150

Reply:
Already contributing to other Swift projects, very nice!!

Reply:
They've merged the PR so we could point to their main branch (if we're not already -- I'm not sure what shows up in Package.resolved). Unfortunately it hasn't been released so we can't point to a version number yet.

Reply:
hmm the only problem is that we'd be pointing to a branch that is changing all the time.